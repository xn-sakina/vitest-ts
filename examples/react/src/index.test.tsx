import React from 'react'
import { Page } from './'
import renderer from 'react-test-renderer'

test('page component', () => {
  const component = renderer.create(<Page extra="test" />)
  const tree = component.toJSON()

  expect(tree).toMatchSnapshot()
  // @ts-ignore
  expect(tree!.children[0].children.includes('test')).toBeTruthy()
})
