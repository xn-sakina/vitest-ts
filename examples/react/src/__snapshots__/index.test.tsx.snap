// Vitest Snapshot v1

exports[`page component 1`] = `
<div>
  <span>
    page: 
    test
  </span>
</div>
`;
